builder/lxd/step_lxd_launch.go

```diff
@@ -58,6 +58,10 @@ func (s *stepLxdLaunch) Cleanup(state multistep.StateBag) {
 	config := state.Get("config").(*Config)
 	ui := state.Get("ui").(packersdk.Ui)
 
+	if config.SkipPublish {
+		return
```

Comment:
What do you think about adding a log here as you did for publication? Just to inform the user about this step being skipped because it wasn't published.

Reply:
I don't think I need to add a log.
Because it outputs the same lines if added here.